src/Apps/Order/Routes/__tests__/Utils/TestPage.tsx

```diff
@@ -0,0 +1,178 @@
+import { Button } from "@artsy/palette"
+import { ArtworkSummaryItemFragmentContainer } from "Apps/Order/Components/ArtworkSummaryItem"
+import { ConditionsOfSaleDisclaimer } from "Apps/Order/Components/ConditionsOfSaleDisclaimer"
+import { CreditCardSummaryItemFragmentContainer } from "Apps/Order/Components/CreditCardSummaryItem"
+import { OrderStepper } from "Apps/Order/Components/OrderStepper"
+import { ShippingSummaryItemFragmentContainer } from "Apps/Order/Components/ShippingSummaryItem"
+import { TransactionDetailsSummaryItem } from "Apps/Order/Components/TransactionDetailsSummaryItem"
+import { ConnectedModalDialog } from "Apps/Order/Dialogs"
+import { ModalButton, ModalDialog } from "Components/Modal/ModalDialog"
+import { Stepper } from "Components/v2"
+import { CountdownTimer } from "Components/v2/CountdownTimer"
+import { MockBoot, renderRelayTree } from "DevTools"
+import { ReactWrapper } from "enzyme"
+import React from "react"
+import { GraphQLTaggedNode } from "react-relay"
+import { flushPromiseQueue } from "Utils/flushPromiseQueue"
+import { Breakpoint } from "Utils/Responsive"
+
+function expectOne<T>(component: ReactWrapper<T>): ReactWrapper<T> {
+  if (component.length !== 1) {
+    // put this behind an if statement to prevent messing up assertion numbers
```

Comment:
Have a look at the tests for whether or not submitting a mutation causes the button's loading spinner to show. There are three assertions visible in those tests and we explicitly call `expect.assertions(3)` to make sure that they were all called.

If this `expectOne` function used `expect` in the happy path, then it would add extra assertions to that total and the test would fail unless we change `3` to `7` which would be mega confusing.